### ISSUE TITLE
Add a link in tool type for local commits

### ISSUE THREAD

**Issue by AlexSmarandache15** [oxygenxml/oxygen-git-plugin#145]
![image](https://user-images.githubusercontent.com/68815668/130433185-3d25dfca-6612-4e98-87e1-23aacd60dd20.png)
Maybe we add a link to the remote branch and to the local branches.

**Comment by sorincarbunaru** [comment]
I'm not sure I fully understand... I think all we should do is to add a label such as `Upstream branch: origin/main`.

**Comment by raducoravu** [comment]
Right now for local branches there is no "Upstream branch " on the tooltip, we should add it right? To show in the tooltip to what upstream branch it connects.

**Comment by sorincarbunaru** [comment]
Yes, it makes sense. But not as a link, but as a plain label, IMO.
